src/demangle.h

```diff
@@ -6,15 +6,19 @@
 #ifndef _DEMANGLE_H
 #define _DEMANGLE_H
 
+#include "rust-demangle.h"
 
 class Demangle {
   private:
     static char* demangleCpp(const char* s);
-    static char* demangleRust(const char* s, const char* e);
+    static char* demangleRust(const char *s, struct demangle const *demangle, bool full_signature);
```

Comment:
I will not ask you to restyle `rust-demangle*` sources (maybe, except for filenames), since they came from a different project, but please don't change coding style of the existing sources.